components/public-api/gitpod/experimental/v1/tokens.proto

```diff
@@ -88,8 +88,11 @@ message ListPersonalAccessTokensResponse {
 }
 
 message RegeneratePersonalAccessTokenRequest {
+    // id is the ID of the PersonalAccessToken
+    string id = 1;
```

Comment:
This performs a field re-ordering which would normally not be backwards compatible, but because we're not using the RPC anywhere yet, this is safe.